cmd/sentinel/state.go

```
@@ -0,0 +1,157 @@
+	}
+
+	switch sub {
+	case "process":
+		return stateProcess(store)
+	case "history":
+		n := 5
+		if len(args) > 1 {
+			v, err := strconv.Atoi(args[1])
+			if err != nil || v < 0 {
+				fmt.Fprintln(os.Stderr, "sentinel state history: n must be a non-negative integer")
+				return 64, nil
```

golangci-lint (2.12.2)

[error] error is not nil (line 75) but it returns nil

(nilerr)